components/Buttons/src/Theming/MDCFloatingButton+MaterialTheming.m

```diff
@@ -45,8 +44,13 @@ - (void)applySecondaryThemeWithScheme:(id<MDCContainerScheming>)scheme {
   [self setElevation:MDCShadowElevationNone forState:UIControlStateDisabled];
 }
 
-- (void)applySecondaryThemeWithColorScheme:(id<MDCColorScheming>)scheme {
-  [MDCFloatingButtonColorThemer applySemanticColorScheme:scheme toButton:self];
+- (void)applySecondaryThemeWithColorScheme:(id<MDCColorScheming>)colorScheme {
+  [self resetUIControlStatesForButtonTheming];
```

Comment:
Please do not change Active libraries as part of this change – this PR should only be adding deprecation warnings. These changes should be a part a future PR to remove the class being deprecated in this PR.

Reply:
Jeff's PR deprecating the ShapeThemer extension did more than just add the deprecation warnings, FWIW: https://github.com/material-components/material-components-ios/pull/8697/files#diff-0c73d1e1b12ea2c632916444f6d375b5L53-R58

Reply:
That's fair, but the PR you linked to is somewhat inconsistent with its use of warning silencing and replacing functionality. These changes are generally safer and easier to review if the replacement and removal of a deprecated API happens in one PR.

Reply:
I'm OK unblocking this CL, but in the future, please restrict deprecation PRs to strictly adding deprecation warnings and silencing those warnings as necessary.

Reply:
Sure